packages/terra-table/src/SelectableTableRows.jsx

```diff
@@ -62,12 +103,38 @@ class SelectableTableRows extends React.Component {
 
   wrappedOnKeyDownForRow(row, index) {
     const initialOnKeyDown = row.props.onKeyDown;
-
     return (event) => {
       if (event.nativeEvent.keyCode === KeyCode.KEY_RETURN || event.nativeEvent.keyCode === KeyCode.KEY_SPACE) {
         // The default isSelectable attribute is either undefined or true, unless the consumer specifies the row's isSelectable attribute as false.
         if (row.props.isSelectable !== false) {
           this.handleOnChange(event, index);
+
+          const { liveRegion } = this.props;
```

Comment:
Same comments for this function.

Reply:
https://github.com/cerner/terra-core/pull/2331/commits/3fb9b80ad6657aec97c4c26e072c7f421c0e62fc